app/controllers/clearance/sessions_controller.rb

```diff
@@ -21,7 +21,11 @@ def destroy
   end
 
   def new
-    render template: 'sessions/new'
+    if current_user
+      redirect_to redirect_url
+    else
+      render template: 'sessions/new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.